packages/moonstone/Panels/Header.js

```diff
@@ -258,7 +270,9 @@ const HeaderBase = kind({
 			case 'dense':
 			case 'standard': return (
 				<Layout component="header" aria-label={title} {...rest} orientation="vertical">
-					{titleOrInput}
+					<Cell component={Transition} type="slide" visible={!minimized} size={minimized ? '0' : titleHeight}>
```

Comment:
Why "slide" and not "clip"? Won't that mean that the Header will only slide out of the way, not changing the DOM dimensions? You may need to add children to verify they move during/after the transition.

Reply:
Using "slide" for the animation and the dimension is set by setting size in Cell